notebooks/2_PrincipalCovariatesRegression.ipynb

```diff
@@ -194,8 +201,8 @@
     "    + (1 - \\alpha) {\\hat{\\mathbf{Y}} \\hat{\\mathbf{Y}}^T},\n",
     "\\end{equation}\n",
     "\n",
-    "such that one can\n",
-    "write $\\rho = \\operatorname{Tr}\\left(\\tilde{\\mathbf{T}}^T\\mathbf{\\tilde{K}}\\tilde{\\mathbf{T}}\\right)$, that is maximized when $\\tilde{\\mathbf{T}}$ contains the principal eigenvectors of $\\mathbf{\\tilde{K}}$ (i.e., the eigenvectors associated with the largest eigenvalues). \n",
+    "Since the trace of the matrix is invariant under cyclic permutation ($\\operatorname{Tr}(\\mathbf{A B C}) = \\operatorname{Tr}(\\mathbf{B C A})$), we can write \n",
```

Comment:
```suggestion
    "Since the trace of the matrix is invariant under cyclic permutation ($\\operatorname{Tr}(\\mathbf{A B C}) = \\operatorname{Tr}(\\mathbf{B C A})  = \\operatorname{Tr}(\\mathbf{C A B})$), we can write \n",
```